pkg/controller/cloudlet_api.go

```diff
@@ -983,23 +1049,24 @@ func (s *CloudletApi) UpdateCloudlet(in *edgeproto.Cloudlet, inCb edgeproto.Clou
 	if err != nil {
 		return err
 	}
-	if !ignoreCRMState(cctx) {
```

Comment:
Is this check not needed anymore?

Reply:
Yeah, I couldn't think of why this check was here, it didn't make any sense. I tried looking through the history, I think a bunch of the CRM upgrade code used to be bundled under this check, but it got moved out and only the access vars update was left behind. Does it make sense to you?

Reply:
makes sense. but if feels like the check is there not to update accessvars in the vault if IgnoreCrm is set. I don't think this matters either way, as CRM is not in the picture here, so yeah this checks looks like it's not doing anything.